### ISSUE TITLE
Error: The render was aborted by the server without a reason.

### ISSUE THREAD

**Issue by nicitaacom** [vercel/next.js#56699]
### Link to the code that reproduces this issue

https://codesandbox.io/p/github/nicitaacom/19_spotify-clone/00-00

### To Reproduce

Link to the code that reproduces this issue
To Reproduce
image

codesandbox - https://codesandbox.io/p/github/nicitaacom/19_spotify-clone/00-00
github repository - https://github.com/Nicitaa/19_spotify-clone

Current vs. Expected behavior
Current behavior
I have this error

Expected behavior
No error at all (because it useless error - it tells me nothing to do or how to fix it - so that's spam)
Or something that fix this error and as result issue will not appears again

Verify canary release
 I verified that the issue exists in the latest Next.js canary release
Provide environment information
Operating System:
  Platform: linux
  Arch: x64
  Version: #1 SMP PREEMPT_DYNAMIC Sun Aug  6 20:05:33 UTC 2023
Binaries:
  Node: 16.17.0
  npm: 8.15.0
  Yarn: 1.22.19
  pnpm: 7.1.0
Relevant Packages:
  next: 13.5.5-canary.5
  eslint-config-next: 13.5.4
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.2.2
Next.js Config:
  output: N/A
Which area(s) are affected? (Select all that apply)
CLI (create-[next-app](https://linear.app/vercel/issue/NEXT-app))

Additional context
I got this error I was writing spotify clone by this guide - https://www.youtube.com/watch?v=2aeMRB8LL4o&t=12065s&ab_channel=CodeWithAntonio

### Current vs. Expected behavior

dsfds

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
12esd
```

### Which area(s) are affected? (Select all that apply)

CLI (create-next-app)

### Additional context

dfs

**Comment by ADTC** [comment]
I'm getting this error whenever I'm editing source code files, such as SCSS modules. Instead of hot-patching the CSS into the page, it cold-reloads the entire page after throwing this error in console.

Sometimes the hot-patching works, but most of the time it's cold-reloading. It's really annoying when you're trying to develop.

**Comment by eric-burel** [comment]
Hi, I will try to create a repro or a more specific bug ticket but I feel like this issue might be related to incremental rendering breakage. In 13.5.4 and 13.5.5 (didn't test earlier version), if I quickly refresh a static page with a 10 second revalidation (pressing F5 like crazy), for a few seconds, it will stop revalidating and serve me a cached version. I don't think there is rate limiting built-in Next? And the only suspicious message I saw is this render abort issue.

**Comment by nicitaacom** [comment]
> Hi, I will try to create a repro or a more specific bug ticket

About repo - its no repo to reproduce it - this issue happens randomly
What do you mean by specific bug ticket?

**Comment by eric-burel** [comment]
@nicitaacom yeah in the end it was not related, my bug is closer to #57039 and #56018, eventhough I also have the random "render aborderted" messages but they don't seem critical

**Comment by nicitaacom** [comment]
> messages but they don't seem critical

I agree with you

I have a bit another case when I already have 1-2 these errors and when I ignore it my terminals turns a mass of bunch of errors

**Comment by dev-abuhasan** [comment]
> @nicitaacom yeah in the end it was not related, my bug is closer to #57039 and #56018, eventhough I also have the random "render aborderted" messages but they don't seem critical

well I have same issue on version next 13.5.6, if I refresh or change pages like creazy, then get this error....

**Comment by matheusdamiao** [comment]
Happening here too. Using next 13.5.4 I got this error and then the page is not rendered completly.

**Comment by nicitaacom** [comment]
This issue is relevent and I think that it's worth looking into it for Next.js developers

If needed I can provide you envs for SUPABASE (or use yours)
And I can get you access - so you can update to latest version of Next.js
So you make sure that it's relevent to current next.js version

**Comment by marcoshernanz** [comment]
Thanks for the report. I reproduced this on 13.5.5-canary.5 as a client-aborted RSC stream. The original error has been suppressed since Next.js 14. The current client-abort logging regression is tracked with an updated reproduction in #96704 and an active fix in #96715, so this report no longer has an actionable current reproduction. Closing in favor of those.
